Elevel/Elevel.Api/Startup.cs

```diff
@@ -28,14 +38,63 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            //Configuration from AppSettings
+            services.Configure<TokenConfiguration>(Configuration.GetSection("JWT"));
+            //User Manager Service
+            services.AddIdentity<ApplicationUser, IdentityRole>().AddEntityFrameworkStores<ApplicationDbContext>();
```

Comment:
here should be IdentityRole<Guid> I think

Reply:
Done.